modules/nupac.nu

```diff
@@ -163,23 +163,23 @@ def remove-from-config [
 # actual package installation happens here
 def install-package [
     package: record
-    --add-to-config(-a): bool
+    add-to-config: bool
```

Comment:
such change allows us to remove `if` statement and then 2 almost the same `each` calls in `nupac install`

hope it doesn't break readability for you